vnext/Shared/Networking/WinRTHttpResource.cpp

```diff
@@ -146,9 +155,28 @@ IAsyncOperation<HttpRequestMessage> WinRTHttpResource::CreateRequest(
       auto file = co_await StorageFile::GetFileFromApplicationUriAsync(Uri{to_hstring(data["uri"].asString())});
       auto stream = co_await file.OpenReadAsync();
       content = HttpStreamContent{std::move(stream)};
-    } else if (!data["form"].empty()) {
-      // #9535 - HTTP form data support
-      // winrt::Windows::Web::Http::HttpMultipartFormDataContent()
+    } else if (!data["formData"].empty()) {
+      winrt::Windows::Web::Http::HttpMultipartFormDataContent multiPartContent;
+      auto formData = data["formData"];
+
+      for (auto &formDataPart : formData) {
+        IHttpContent formContent{nullptr};
+        if (!formDataPart["string"].empty()) {
```

Comment:
>formDataPart["string"] [](http://example.com/codeflow?start=13&length=22)

formData's 'auto' decl makes it difficult to understand what the actual type is, but if the index operator here has the same semantics as that of std::unordered_map (which is to insert one of there's no value with the respective key), would it be better to call 'at' or 'contains' here?

Reply:
This object is of type `folly::dynamic`.
It can work as an array or as an unordered map.